app/javascript/components/admin/site_settings/appearance/BrandingImage.jsx

```diff
@@ -1,18 +1,51 @@
 import React from 'react';
-import { Card } from 'react-bootstrap';
-import { CloudUploadIcon } from '@heroicons/react/outline';
+import { Card, Stack } from 'react-bootstrap';
+import { CloudUploadIcon, TrashIcon } from '@heroicons/react/outline';
 import useUpdateSiteSetting from '../../../../hooks/mutations/admin/site_settings/useUpdateSiteSetting';
+import useSiteSettings from '../../../../hooks/queries/admin/site_settings/useSiteSettings';
 import FilesDragAndDrop from '../../../shared_components/utilities/FilesDragAndDrop';
+import Logo from '../../../shared_components/Logo';
+import Modal from '../../../shared_components/modals/Modal';
+import DeleteBrandingImageModal from './DeleteBrandingImageModal';
 
 export default function BrandingImage() {
   const updateSiteSetting = useUpdateSiteSetting('BrandingImage');
+  const { data: siteSettings } = useSiteSettings();
 
-  return (
-    <div className="mt-4">
-      <h6> Branding Image </h6>
+  if (siteSettings.BrandingImage.length > 1) {
```

Comment:
Why `siteSettings.BrandingImage.length > 1`?
An empty string is a falsy value in JS,